app/api/v1/models.py

```diff
@@ -0,0 +1,97 @@
+import random
+
+users = []
+parcels = []
+
+
+class UserModel:
+    """This is the model class to manipulate data"""
+    def __init__(self):
+        """initialzation for our data"""
+        self.users = users
+
+    def add_user(self, data):
+        """this adds users to our dict"""
+        user = self.users
+        user_id = len(user) + 1
```

Comment:
Let's use `UUID` to get unique ids. You can refer to its documentation [here](https://docs.python.org/3/library/uuid.html#uuid.uuid4).
Use this also for your parcel's order number.

Reply:
Thanks have been having problem getting the unique ID, Ill go through the document and implement it .

Reply:
Implemented and used UUID for the user key and parcel